manifests/charts/gateways/istio-egress/templates/_affinity.tpl

```diff
@@ -11,6 +11,7 @@ nodeAffinity:
 {{- define "nodeAffinityRequiredDuringScheduling" }}
       nodeSelectorTerms:
       - matchExpressions:
+        {{- if .global.arch }}
```

Comment:
This change comes from upstream [bug fix](https://github.com/maistra/istio/commit/361ab420c06ea833b05d50367375cc16d7ef98bc) which overlaps with [our change](https://github.com/maistra/istio/commit/f418472a1c3f5aa8efc06859df18da8530458995) of this template, so we could revert our commit before merging this PR to avoid confusion during the next rebase.

Reply:
After a short reflection, I think we can ignore it in this release. I will just prepare a note for the next rebase, because it's not the only one commit to skip.